godel_path_execution/src/path_execution_service_node.cpp

```diff
@@ -0,0 +1,29 @@
+#include <godel_path_execution/path_execution_service.h>
+
+#include <ros/ros.h>
+
+//"joint_trajectory_action"
+
+const static std::string DEFAULT_SERVICE_NAME = "path_execution";
+
+int main(int argc, char** argv)
+{
+  ros::init(argc, argv, "path_execution_service_node");
+
+  ros::NodeHandle nh;
+  ros::NodeHandle pnh ("~");
+  std::string action_server_name, service_name;
+
+  if (!pnh.getParam<std::string>("action_server_name", action_server_name))
```

Comment:
This action_name can be reconfigured with rosparam.  I would modify it if it is an easy fix.